express/blocks/hero-animation-beta/hero-animation-beta.css

```diff
@@ -318,6 +322,19 @@ main .hero-animation-beta p.button-container a.secondary-button {
   border-color: #fff;
 }
 
+@media (min-width: 480px) and (max-width: 949px) {
+  main .hero-animation-beta p.button-container a.secondary-button {
```

Comment:
in my opinion this is becoming too complicated. 
instead of doing this, how about setting the secondary button to have that white background color if the browser width is more than 950px. That saves you on code and is much simpler. 
It might also immediately fix the other bug that we're having, that the button "disappears", when you hover over it.

Reply:
this still needs to be simplified imo - if you want we can jump in a call and do it together